pdf/README.md

```diff
@@ -0,0 +1,29 @@
+# PDF Generation of Toolkit
+
+The contents of this folder are used towards the automated generation of a PDF of the Digital Public Goods Toolkit. This is accomplished through the preprocessing and subsequent collating of selected Markdown files to output a single PDF file.
+
+## Requirements
+
+Generating a PDF, requires both `pandoc` (v2.9 or higher) and `TeX` to be installed in the system.
+
+* MacOS: Run the following commands to install the necessary dependencies
+
+	```bash
+	brew install pandoc
+	brew install basictex
+	sudo tlmgr update --self
+	sudo tlmgr install titlesec
+	```
+
+## Generate the PDF
+
+Simply run `make` inside this folder:
+
+```bash
+cd pdf
+make
+```
+
+which will output a file named `publicgoods-toolkit-{version}.pdf`.
```

Comment:
The new version of the Makefile I just reviewed (in PR #70) no longer puts the "`-{version}`" into the filename, I think?  If so, this line of documentation would need to be adjusted accordingly.

Reply:
Correct, updated documentation accordingly in 7c22f79